docker-compose.yml

```diff
@@ -13,14 +13,17 @@ services:
 
   database:
     container_name: storage
-    image: amsterdam/postgres11
+    build: ./docker/database
     platform: linux/amd64
     ports:
     - "5406:5432"
     environment:
       POSTGRES_PASSWORD: insecure
       POSTGRES_DB: gob
       POSTGRES_USER: gob
+    volumes:
+    - "~/.ssh/datapunt.key:/root/.ssh/datapunt.key"
```

Comment:
Is deze eigenlijk echt nodig?

Reply:
Goeie vraag eigenlijk. Ik weet niet precies meer waarom deze gebruikt werd. Heb 'm toen ik deze code wijzigde vast met een reden toegevoegd ook, maar dat is ook al even geleden.. 🙈